ddsc/cmdparser.py

```diff
@@ -291,14 +293,23 @@ def register_list_command(self, list_func):
     def register_delete_command(self, delete_func):
         """
         Add 'delete' command delete a project from the remote store.
-        :param list_func: function: run when user choses this option.
+        :param delete_func: function: run when user choses this option.
         """
         description = "Permanently delete a project."
         delete_parser = self.subparsers.add_parser('delete', description=description)
         add_project_name_arg(delete_parser, help_text="Name of the project to delete.")
         _add_force_arg(delete_parser, "Do not prompt before deleting.")
         delete_parser.set_defaults(func=delete_func)
 
+    def register_list_auth_roles_command(self, list_auth_roles_func):
+        """
+        Add 'list_auth_roles' command to list project authorization roles that can be used with add_user.
+        :param list_auth_roles_func: function: run when user choses this option.
+        """
+        description = "List authorization roles for use with add_user command."
+        delete_parser = self.subparsers.add_parser('list_auth_roles', description=description)
```

Comment:
Looks like `delete_parser` was copied/pasted from previous function and could be renamed